README.md

```diff
@@ -16,6 +16,7 @@
 [![run with conda](http://img.shields.io/badge/run%20with-conda-3EB049?labelColor=000000&logo=anaconda)](https://docs.conda.io/en/latest/)
 [![run with docker](https://img.shields.io/badge/run%20with-docker-0db7ed?labelColor=000000&logo=docker)](https://www.docker.com/)
 [![run with singularity](https://img.shields.io/badge/run%20with-singularity-1d355c.svg?labelColor=000000)](https://sylabs.io/docs/)
+[![Static Badge](https://img.shields.io/badge/%F0%9F%8D%94%20BigMAG-compatible-%231eaf64)](https://github.com/jeffe107/BIgMAG)
```

Comment:
Can you move this to the end of all the badges, otherwise it'll complicate our pipeline template merge every time :) (nice to have though!)